### PR TITLE
[18.0][MIG] sale_resource_booking: Migration to 18.0

### PR DESCRIPTION
**Pull request by Anushapadinharepurakkal**

### CODE REVIEW

**Review by tarteo — APPROVED** [review]
Tested on runboat, seems to work fine. When the booking is created from a sales order the name of the booking is empty but I checked that is also the case in 17

**Comment by tarteo** [comment]
I did some more test, there's one issue where the "Confirmation dialog to autofill resource bookings" wizard is empty if I add a section to the sale order lines.

with section:
image
without:
image

**Review by ByteMeAsap — APPROVED** [review]
LGTM!!

**Comment by ByteMeAsap** [comment]
@pedrobaeza , can you please merge this?

File: sale_resource_booking/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Tecnativa - Jairo Llopis
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sell resource bookings",
```
Comment (rousseldenis) [inline sale_resource_booking/__manifest__.py:5]:
```suggestion
    "name": "Sale resource bookings",
```

Comment (pedrobaeza) [reply sale_resource_booking/__manifest__.py:5]: Well, in fact the original intention of the description is to put "sell"

**Comment by Anushapadinharepurakkal** [comment]
@pedrobaeza 

Can you please merge this?

**Comment by pedrobaeza** [comment]
Waiting for @rousseldenis' feedback

**Review by rousseldenis — APPROVED** [review]

**Comment by Anushapadinharepurakkal** [comment]
@rousseldenis 

Can you please merge this?

**Comment by ByteMeAsap** [comment]
@pedrobaeza

Can you please merge this now?
